src/index.html

```diff
@@ -11,12 +11,106 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__content">
+        <a
+          href="#"
+          class="logo-link"
+        >
+          <img
+            class="logo-img"
+            src="./images/logo.png"
+            alt="company-logo"
+          />
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                class="nav__link is-active"
+                href="#apple"
+              >
+                Apple
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#samsung"
+              >
+                Samsung
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                href="#smartphones"
+              >
+                Smartphones
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                data-qa="hover"
+                href="#laptops&computers"
```

Comment:
The `href` attribute value `#laptops&computers` contains an ampersand (`&`) which is not a valid character in a URL fragment identifier. Consider replacing it with a hyphen (`-`) or encoding it as `%26`.